projects/vdk-plugins/airflow-provider-vdk/vdk_provider/example_dags/example_vdk.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from datetime import datetime
+
+from airflow import DAG
+from vdk_provider.operators.vdk import VDKOperator
+
+with DAG(
+    "example_vdk",
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
+    catchup=False,
+    tags=["example"],
+) as dag:
+    op1 = VDKOperator(
+        conn_id="example_vdk_connection",
+        job_name="example_vdk_job1",
+        team_name="team_name",
+        task_id="job1",
```

Comment:
what is task_id used for?

Reply:
We need to think about how to generate it - `job_name + something`.

Reply:
I think task_id is supposed to be chosen by a user writing a DAG - we could probably generate it automatically but I'm not sure it's intended to be.